src/components/AppBar/AppBar.js

```diff
@@ -18,27 +19,32 @@ const styles = () => ({
   },
 });
 
-const RVAppBar = (props) => {
+const RVAppBar = props => {
   const { classes } = props;
 
   return (
     <AppBar position="static" color="default" className={classes.appBar}>
-      <Toolbar>
-        <Typography
-          variant="h6"
-          color="inherit"
-          noWrap
-          className={classes.toolbarTitle}
-        >
-          Les fieres
-        </Typography>
-        <Button>Features</Button>
-        <Button>Enterprise</Button>
-        <Button>Support</Button>
-        <Button color="primary" variant="outlined">
-          Login
-        </Button>
-      </Toolbar>
+      <BrowserRouter>
```

Comment:
You should remove <BrowserRouter> from here. We have a <BrowserRouter> on the App.js. This will make the <Link>'s interact with this Router, preventing the top one to work at all... We should only have one <BrowserRouter> in the app